config/application.rb

```diff
@@ -41,7 +41,5 @@ class Application < Rails::Application
     )
 
     config.autoload_paths += Dir[Rails.root.join('app', '**/')]
-
-    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
@tansaku I removed this configuration since is not supported anymore. Do you know some drawback from removing it?